### PR TITLE
fix: change dockerfile to being able to set keep alive timeout

### PR DESCRIPTION
**Pull request by mikitabut**

### CODE REVIEW

**Comment by mikitabut** [comment]
/deploy-review
>Deployment status: success

**Comment by mikitabut** [comment]
/deploy-review
>Deployment status: failed

**Comment by mikitabut** [comment]
/deploy-review
>Deployment status: failed

**Comment by mikitabut** [comment]
/deploy-review
>Deployment status: success

File: startup.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+KEEP_ALIVE_TIMEOUT="${KEEP_ALIVE_TIMEOUT:-5000}"
+
+echo KEEP_ALIVE_TIMEOUT
```
Comment (IlyaBondar) [inline startup.sh]:
```suggestion
echo "KEEP_ALIVE_TIMEOUT=$KEEP_ALIVE_TIMEOUT"
```

**Comment by mikitabut** [comment]
/deploy-review
>Deployment status: success

File: startup.sh (hunk 1 above)
Comment (mikitabut) [reply startup.sh]: Not actual anymore

**Review by IlyaBondar — APPROVED** [review]

**Pull request merged**